### ISSUE TITLE
[Screen Reader – Make Code – About] Purpose of '3.0.17, 6.0.18, v2.2.0-rc6' version link controls are not descriptive under 'About' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3629]
**User Experience:**
If the purpose of each link is not determined from the link text alone or from the link text together with its programmatically determined link context, it would be ambiguous to all users in general.  Screen reader user will be affected if screen reader doesn't provide descriptive information for any control as user won't get the purpose of such control.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "About" option and select it.
7. Navigate through all the controls appearing under "About" dialog.
8. Verify that '3.0.17, 6.0.18, v2.2.0-rc6' version link controls by using screen reader.

**Actual Result:**
Purpose of '3.0.17, 6.0.18, v2.2.0-rc6' version link controls are not descriptive under 'About' pop-up window. Screen reader is reading only digit of version.

**Expected Result:**
Purpose of '3.0.17, 6.0.18, v2.2.0-rc6' version link controls should be descriptive under 'About' pop-up window and screen reader should be read as 'makecode.microbit.org version:  3.0.17 link, Microsoft MakeCode version:  6.0.18 link, microbit runtime version:  v2.2.0-rc6'.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 2.4.4 - Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox, Chrome
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.4, #Win10-Edge(Chromium)

[WCAG2.4.4_About.zip](https://github.com/microsoft/pxt-microbit/files/5509731/WCAG2.4.4_About.zip)
